yaswanth1701/benchmark
# Added post processing script for benchmarks

Test on: Ubuntu 22 and gazebo 11
This is a follow-up PR to #27  and solves  #25 . Now instead of calculating error/metric in gazebo api a separate python script has been added.  This facilitates simulator-independent metric calculation functionality. The script currently processes the logged data in CSV format for calculations.
To test:
```bash
python3 post-processing.py
```
Note: Make sure to add results to the appropriate folders to avoid errors. One would require gz-math7 python binding which can be installed by running.
```bash
sudo apt-get install python3-gz-math7
```
